bloom-housing/bloom
# fix: add missing translations

# Pull Request Template

## Issue Overview

Pulls the missing translation fixes back to core from Doorway, which includes these two PRs:
https://github.com/metrotranscom/doorway/pull/606
https://github.com/metrotranscom/doorway/pull/607

## Description

All pulling from the above:

Removes strings we are no longer using in core (mostly program and preference strings).

Re-alphabetizes the list (causes more diff).

Adds the generate-missing-translations script from Doorway which can call the google translate API for a particular language for a CSV of missing strings.

Pulled over translations we already got from Doorway, which included adding into core some of Doorway's strings, and then ran the script for the rest.

Will add the job in a different PR!

## How Can This Be Tested/Reviewed?

Confirmed with product that it's okay to add the machine translations into core. You can feel free to run the missing translations job locally to see it succeed.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
